ee/api/conversation.py

```diff
@@ -35,15 +38,39 @@ def validate(self, data):
         return data
 
 
-class ConversationViewSet(TeamAndOrgViewSetMixin, GenericViewSet):
+class ConversationSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Conversation
+        fields = "__all__"
```

Comment:
Let's be explicit with the fields, because that way we can improve in two ways:
1. We can omit `team_id`, which would just be duplicated from the endpoint path
2. We can explicitly make all the fields read-only (I know the endpoint doesn't have PATCH, but it will be much harder to accidentally allow something wrong in the future, for free)
```suggestion
        fields = ["user_id", "status", "type", "messages"]
        read_only_fields = fields
```
Also this exposes that we're including `user_id`, which in itself _probably_ won't be useful on the frontend, in that `UserBasicSerializer` will likely be way easier to consume in practice.

Reply:
Reasonable! I've updated the fields.